gui/src/vmm/cpu/controller.rs

```diff
@@ -30,3 +32,7 @@ impl Drop for CpuController {
         unsafe { ManuallyDrop::take(&mut self.thread).join().unwrap() };
     }
 }
+
+pub(super) enum ResumeAction {
+    Continue,
+}
```

Comment:
I don't think we need this. According to the docs of `MultiThreadResume::resume`:

```rust
    /// Any thread that wasn't explicitly resumed by a `set_resume_action_XXX`
    /// method should be resumed as though it was resumed with
    /// `set_resume_action_continue`.
```

That mean we don't even to do anything in `clear_resume_actions` and `set_resume_action_continue`.